public/app/features/explore/Logs/Logs.tsx

```diff
@@ -370,6 +371,17 @@ class UnthemedLogs extends PureComponent<Props, State> {
   };
 
   scrollIntoView = (element: HTMLElement) => {
+    if (config.featureToggles.exploreScrollableLogsContainer) {
+      this.scrollToTopLogs();
+      if (this.logsContainer.current) {
+        this.logsContainer.current.scroll({
+          behavior: 'smooth',
+          top: this.logsContainer.current.scrollTop + element.getBoundingClientRect().top - window.innerHeight / 2,
+        });
+      }
+
+      return;
+    }
     const { scrollElement } = this.props;
```

Comment:
```suggestion
    } else {
    const { scrollElement } = this.props;
```

Personal preference, but I would rather do `if/else` than returning undefined.

Reply:
> than returning undefined.

There is no difference if a function in JavaScript calls `return;` or just nothing.

```js
function a(){}
typeof a()  === 'undefined' // -> true

// -------------

function b(){
return;
}
typeof b() === 'undefined' // -> true
```

Reply:
I know, just personally it is easier for me to mentally process
```
if (a === something) {
do something
} else {
do something else
}
```

than 

```
if (a === something) {
do something
return
} 

do something else
```

So as I sad, just personal preference 🙂